src/tsickle.ts

```diff
@@ -785,6 +785,17 @@ class Annotator extends ClosureRewriter {
       case ts.SyntaxKind.AsExpression:
         // Both of these cases are AssertionExpressions.
         const typeAssertion = node as ts.AssertionExpression;
+        if (this.polymerBehaviorStackCount > 0) {
+          // Don't emit type casts for Polymer behaviors that are declared
+          // by calling the Polymer function
+          // as the Polymer closure plugin does not work when emitting them.
```

Comment:
We are doing the right thing. However, the polymer closure plugin does not work when we (correctly) emit type comments for casts for values in the `behaviors` property of an object literal that is passed to `Polymer`. 

The non transformer version never emitted the type cast (or rather typescript removed it due to the bug in Typescript with comments).

This needs to be fixed in Polymer, I am just adding this so we don't depend on them for landing our transformer support.